client/commands/initialize_pysa.py

```diff
@@ -73,9 +73,8 @@ def _setup_environment() -> None:
     run_infer = log.get_yes_no_input("Would you like to generate type annotations?")
     if run_infer:
         subprocess.run(["pyre", "infer", "-i"], check=True)
-    pyre_check_path = find_taint_models_directory()
-    if pyre_check_path and os.path.isdir(pyre_check_path / "pyre_check"):
-        pyre_check_path = pyre_check_path / "pyre_check"
+    pyre_check_path = find_pyre_directory()
+    if pyre_check_path is not None:
```

Comment:
I think we should call `sapp filter` with `lib/pyre_check/pysa_filters` instead of `lib/pyre_check`.
Could you update the code to do that? We could have a `find_pysa_filters_directory()`